index.html

```diff
@@ -409,8 +409,9 @@
 				<li>Otherwise, fire the event to the object returned by normal hit test mechanisms (out of scope for this specification).</li>
 			</ul>
             <h4>Process Pending Pointer Capture</h4>
-            <p>Whenever a user agent is to fire a Pointer Event that is not <code>gotpointercapture</code> or <code>lostpointercapture</code>, it must first run these steps:</p>
+            <p>Whenever a user agent is to fire a Pointer Event that is not <code>gotpointercapture</code> or <code>lostpointercapture</code>, it must first run these steps for the corresponding <code>pointerId</code>:</p>
```

Comment:
The opening para need to include the immediate processing during set/release capture, something like this:
> Calls to <code>setPointerCapture</code> or <code>releasePointerCapture</code> may also trigger these steps, see Sec10.1 and 10.2 for details.